src/ProjectService.Business/Commands/File/RemoveFilesCommand.cs

```diff
@@ -93,16 +48,13 @@ public async Task<OperationResultResponse<bool>> ExecuteAsync(RemoveFilesRequest
 
       OperationResultResponse<bool> response = new();
 
-      bool result = await RemoveFilesAsync(request.FilesIds, response.Errors);
+      response.Body = await _repository.RemoveAsync(request.FilesIds);
 
-      if (!result)
+      if (response.Body)
       {
-        return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.BadRequest, response.Errors);
+        await _publish.RemoveFilesAsync(request.FilesIds);
       }
```

Comment:
add else and set response = responseCreator......

Reply:
added